docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -0,0 +1,472 @@
+= Vector Collection
+:page-enterprise: true
+:page-beta: true
+
+== Intro
+
+The primary object for interacting with the vector storage is a Vector Collection. This collection holds information about the vectors and associated metadata (user value).
+
+A collection consists of one or more indexes, all sharing a common metadata storage. Each index represents a distinct vector space. Vectors from different indexes are stored independently, and indexes can have different configurations.
+
+Conceptually, a vector collection resembles a key-value store. Here, the key is a user-defined unique identifier, and the value is an object containing metadata (user value) alongside several vectors — one for each index.
+
+Users are permitted to store any necessary value for work as metadata (user value). This can be additional characteristics of the source data or even the source data itself.
+
+== Configuration
+
+Collection configuration can be set dynamically during vector collection creation or statically during cluster configuration. Unlike other data structures, the configuration must be set up before the collection can be used. Below is a table listing all available options:
+
+.Collection configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector collection.
+|Y
+|null
+
+|indexes
+|Information about indexes configuration
+|Y
+|null
+|===
+
+.Index configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector index.
+|`N` - for single-index vector collections. `Y` - for multi-index collection
+|null
+
+|dimension
+|Vectors dimension
+|Y
+|null
+
+|metric
+|Distance metric used to calculate the distance between two vectors.
+See table <<available-metrics, Available distance metrics.>>
+|Y
+|null
+
+|max-degree
+|The maximum number of neighbors per node is calculated as max-degree * 2
+|Y
+|16
+
+|ef-construction
+|The size of the search queue to use when finding nearest neighbors.
+|Y
+|100
+
+|use-deduplication
+|The option for vector deduplication.
+When disabled, a new node will be created for each vector, even if it is the same as an existing one.
```

Comment:
I think that this description may be hard to understand, "node" is not defined in the docs.
we should add more info about pros and cons of using and not using deduplication. I can draft a section about it.

Reply:
7e1b5c3452fd6c95ac9cd6d02014415617138c44

Reply:
important difference is also that without deduplication in some rare cases some vectors may become unreachable. BTW, does `optimize` fix it?